pyspark-k8s/Dockerfile

```diff
@@ -39,9 +39,24 @@ ENV SPARK_HOME=/stackable/spark
 ENV PATH=$SPARK_HOME:$PATH:/bin:$JAVA_HOME/bin:$JAVA_HOME/jre/bin:$HOME/.local/bin
 ENV PYSPARK_PYTHON=/usr/bin/python
 ENV PYTHONPATH=$SPARK_HOME/python:$PYTHONPATH
+ENV TINI_VERSION=v0.19.0
+
+# Ugly to download both archs here, there isn't a decent way to export an environment variable or have an if-else switch 
+ADD https://github.com/krallin/tini/releases/download/${TINI_VERSION}/tini-arm64 spark-k8s/stackable/utils/tini-arm64
```

Comment:
I don't think we pull anything directly from github in the other Dockerfiles, so we probably shouldn't do it here either: download the library to our repo and fetch it from there, to remove a dependency on github availability.

Reply:
Good point! Changed that